grype/pkg/syft_sbom_provider.go

```diff
@@ -24,17 +24,17 @@ func syftSBOMProvider(userInput string) ([]Package, Context, error) {
 		return nil, Context{}, err
 	}
 
-	catalog, srcMetadata, theDistro, _, formatOption, err := syft.Decode(reader)
+	sbom, formatOption, err := syft.Decode(reader)
```

Comment:
I'm guessing the syft update was to return a data structure instead of individual components